bpf/flows.c

```diff
@@ -55,6 +54,13 @@
 #define FIN_ACK_FLAG 0x200
 #define RST_ACK_FLAG 0x400
 
+struct sctphdr {
```

Comment:
Could move this declaration be moved to flow.h

Reply:
this structure used only internally by flow.c not need to be exported outside